Corona-Warn-App/src/main/java/de/rki/coronawarnapp/http/RetryInterceptor.kt

```diff
@@ -0,0 +1,23 @@
+package de.rki.coronawarnapp.http
+
+import android.util.Log
+import okhttp3.Interceptor
+import okhttp3.Request
+import okhttp3.Response
+
+class RetryInterceptor : Interceptor {
```

Comment:
AFAIK [OkHttp](https://square.github.io/okhttp/4.x/okhttp/okhttp3/-ok-http-client/-builder/retry-on-connection-failure/) has implicit retry policy, so no need for that interceptor (unless you disable the built-in retry policy).

Reply:
This is an interceptor for every request failure that is not catches by OkHTTP:
Unreachable IP addresses. Stale pooled connections. Unreachable proxy servers.

This means Server Errors are not covered.